### DISCUSSION TITLE
Somehow "/listings" is pointing to "/home/bagatooc/public_html/listings"

### DISCUSSION THREAD

**Discussion by MvdO79** [wheels-dev/wheels#1399] [Q&A] [answered]
Somehow "/listings" is pointing to "/home/bagatooc/public_html/listings". Locally this problem doesn't ocure, remote it does.

The route is:

`.get(name="listings", pattern="listings/[page]", to="listings##listings")`

I also see this problem when using "/listings" in a image path like "images/listings". Then also the "/listings" path is replaced by "/home/bagatooc/public_html/listings" resulting in a image path like "images/home/bagatooc/public_html/listings".

I hope this description rings a bell by any of you. Maybe it is not Cfwheels related but hosting related? Lucee related? No idea!

**Comment by dbelanger** [comment, marked as answer]
Ideas: If it's on shared hosting, pop an `/index.cfm//listings` in your url to see if you hit the right page.  If you do, you've got a redirect problem in your .htaccess or web.config.  
Also, check the controller of listings##listings to ensure your listing function doesn't have an incorrect view location.

**Comment by MvdO79** [comment]
Thanks! "/index.cfm//listings" was indeed hitting the right page, gonna check on your suggestions.

**Comment by MvdO79** [comment]
Still struggling... but some new insights. All routes are working perfect, accept "/listings".

When I use "/listings/9b1c6/" it works.

When I use "/listings/9b1c6" I get redirected to "/home/bagatooc/public_html/listings/9b1c6/".

When I change the route in something like "/listingsss" it works.

Can anyone agree this has to be a "/listings" mapping somewhere that points to my root?

Also because "/listings" is replaced on all paths with "/home/bagatooc/public_html/listings".

How can I check if and where such mapping exists?

My host doesn't think they have such mapping set so I am clueless...

**Reply by neokoenig** [reply]
Might be something to do with the trailing slash? I've certainly seen URL rewriting redirecting to stupid places because of it.

Either way, I'd look to see if there are any additional htaccess files higher up the chain which could be overriding your local one?

**Reply by MvdO79** [reply]
Thanks @neokoenig, gonna check that.

**Comment by MvdO79** [comment]
So, this isn't a pretty solution but I need to move on... Due to shared hosting I am limited in debugging. Just changed /listings to /ads and all my problems are gone. In the case I can't decide such 'solution' I really wouldn't have known what to do. Thanks all for helping out though!
